Added XTONE Smart Stomp and more docs

galkk opened the pull request:
Added settings for XTONE Smart Stomp and a bit more links to configs/examples. I feel that it will be beneficial for new users to see working examples.

I plan to check my other devices, if they're working: Yamaha THR10 and MOOER GE300, but, most likely, I'll create another PRs for them. Thank you for your great work.

galkk commented:
You can see how it looks like in my repo start page https://github.com/galkk/rs_asio#known-issues

mdias commented:
This is so much needed! Thanks for the effort!
I'll make a suggestion before moving; since we're bound to want to make more of these for different interfaces, this would soon pollute the main readme file, so what I would suggest is having a folder per interface on `docs/` and the guide in there.

For example, for this interface we'd have `docs/xtone-smart-stomp/GUIDE.md` and we'd link to that from the interface list on the main readme. This would make it easy and clean to support many different interfaces. What do you think?

galkk commented:
> I'll make a suggestion before merging; since we're bound to want to make more of these for different interfaces, this would soon pollute the main readme file, so what I would suggest is having a folder per interface on docs/ and the guide in there.

That's actually how I was thinking to do that in the beginning, but thought that this may be an overkill :)

Moved specific interface doc into docs/interface_name and added README.md there to have it rendered by github automatically

mdias commented on README.md:
```diff
@@ -6,6 +6,7 @@ It patches game code at runtime to allow intervening in the process of WASAPI de
 ## How to use
 
 - Copy the contents (`avrt.dll`, `RS_ASIO.dll`, `RS_ASIO.ini`) of [latest release](https://github.com/mdias/rs_asio/releases/latest) (zip arhive release-xxx.zip) to the game folder.
+  - If you're using Steam version of Rocksmith, you can find local folder by right clicking on a Rocksmith in your library, and selecting menu "Manage" -> "Browse local files"
```
We're currently only compatible with Steam version, can you please change the wording to avoid people thinking otherwise?

galkk replied:
done

mdias commented:
I'm very happy with this!
Please tell me when you're done so I can merge :)

galkk commented:
Fixed steam-related wording

mdias commented:
Merged! Thanks a lot for the contribution 👍
